### ISSUE TITLE
ListItem has broken styling

### ISSUE THREAD

**Issue by mendlik** [borela/react-toolbox-plus-router#3]
Hi,

It's me again. I spent some time figuring out why there is no hover effect in `react-toolbox-plus-router/list/ListItem.js`.

Sample:
```
  <List selectable ripple>
    <ListItem caption="About from react-toolbox-plus-router" to="/about" />
    <RtListItem caption="About from react-toolbox" to="/about" />
  </List>
```

After some research I found type comparison in List component https://github.com/react-toolbox/react-toolbox/blob/dev/components/list/List.js#L33 where `selectable` property is passed to children components.

My day job is related with backend on weekends I'm trying to implement a startup idea. ...why is it so hard to do a decent frontend? ;)

**Comment by borela** [comment]
Yeah, I know the pain, I don't like frontend's too lol I believe that to fix that, the ListItem in this lib must inherit the one from the react toolbox, not sure if it would work, I am considering just forking the main project and make all the changes as I need other features like inheriting the "disabled" property  so that I can disable a parent component and all nested children gets disabled too, simpler theming system, etc...
